packages/libs/react-ui/src/components/Notification/NotificationContainer.tsx

```diff
@@ -1,3 +1,5 @@
+import type {
```

Comment:
lint?

Reply:
It's not a type though, is it?

Reply:
It's the linter that's causing that.. I'll add an eslint-disable-next-line